diff -Naur --exclude '*~' TuxMathScrabble/tuxmathscrabble TuxMathScrabble-fix-path/tuxmathscrabble
--- TuxMathScrabble/tuxmathscrabble	2008-11-27 06:34:58.000000000 +0100
+++ TuxMathScrabble-fix-path/tuxmathscrabble	2009-04-13 15:44:09.000000000 +0200
@@ -17,6 +17,7 @@
 ***********************************************************/
 """
 import os,sys,string,time
+sys.path.append('/usr/share/games/tuxmathscrabble/lib')
 from TuxMathScrabble.tms import *
 
 if __name__ == "__main__":
diff -Naur --exclude '*~' TuxMathScrabble/TuxMathScrabble/environment.py TuxMathScrabble-fix-path/TuxMathScrabble/environment.py
--- TuxMathScrabble/TuxMathScrabble/environment.py	2009-01-20 06:16:40.000000000 +0100
+++ TuxMathScrabble-fix-path/TuxMathScrabble/environment.py	2009-04-13 16:02:30.000000000 +0200
@@ -34,9 +34,8 @@
 		
 		if OS=='lin':
 			import pygame 
-			for sitepkgdir in sys.path:
-				if sitepkgdir[-13:]=='site-packages':break
-			configdir=os.path.join('/','var','games',appname)#,os.path.basename(appname)
+			sitepkgdir=os.path.join('/','usr','share','games','tuxmathscrabble', 'lib')
+			configdir=os.path.join('/','usr','share','games','tuxmathscrabble')#,os.path.basename(appname)
 			homedir=os.getenv('HOME')
 		
 		elif OS=='win':
@@ -51,15 +50,6 @@
 			configdir='./'
 			homedir=os.getenv('HOME')
 		
-		
-		#If application hasn't been installed (via setup.py) then run locally:
-		if os.path.exists(os.path.join(sitepkgdir,appname,'tms.py')) and OS=='win':pass
-		else:
-			sitepkgdir='./'
-			configdir='./'
-			homedir=os.getenv('HOME')
-		
-			
 		self.OS=OS
 		self.sitepkgdir=sitepkgdir
 		self.configdir=configdir
